### PR TITLE
Add support for downloading embargoed assets

### PR DESCRIPTION
**Pull request by mugli**
Currently, the exporter cannot download embargoed assets. This PR adds the capability.

### CODE REVIEW

File: lib/utils/embargoedAssets.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+export function isEmbargoedAsset(url) {
+  const pattern = /((images)|(assets)|(downloads)|(videos))\.secure\./
+  return pattern.test(url)
+}
+
+export function createAssetKeyValidity() {
+  // Create a future timestamp with maximum validity, 48h
+  return Date.now() + 48 * 60 * 60 * 1000
```
Comment (leonardofreitass) [inline lib/utils/embargoedAssets.js:82]: I wonder if we should have a smaller TTL here since export should not take too long 🤔

**Review by leonardofreitass — APPROVED** [review]
LGTM, great job!

File: lib/utils/embargoedAssets.js (hunk 1 above)
Comment (mugli) [reply lib/utils/embargoedAssets.js:82]: Agreed, a shorter TTL makes sense. Any suggestion?

File: lib/utils/embargoedAssets.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import rp from 'request-promise'
+import jwt from 'jsonwebtoken'
+
+const SIX_HOURS_IN_MS = 6 * 60 * 60 * 1000
```
Comment (asaaki) [inline lib/utils/embargoedAssets.js:4]: @leonardofreitass during a live review we settled with this value for now.
But we're also okay to lower it further, as the logic is quite smart and would fetch a new token if necessary.
6 hours is mostly a compromise between not too long for potential abuse and not too short that you can still download a huge space in one go.
WDYT?

**Review by asaaki — APPROVED** [review]
`LGTM` :eyes: (we have a live review; code + demo)

Implementation looks solid, the default expiry we can adjust if anyone has concerns (see comment).

**Review by marcolink — APPROVED** [review]
I don't have much domain knowledge about embargoed assets, but from a general "adding things to the CLI" perspective, everything looks fine. Also, great to see all the tests 🚀

**Pull request merged**